### DISCUSSION TITLE
Wix v4 Custom Theme RadioButtons

### DISCUSSION THREAD

**Discussion by ShakeyBones3** [wixtoolset/issues#7230] [Questions] [answered]
I'm trying to create a custom theme for a `bal:WixStandardBootstrapperApplication` that uses a radio buttons to set a variable's value for an install condition. I'm able to do it with check boxes, but when I add RadioButtons to the markup the installer fails to launch. When I look in the `cleanroom.log` file, the only error I'm seeing is `Exit code: 0xc0000005`. The other log file isn't showing any errors / failures. Am I using this incorrectly or is the feature not implemented yet?

```xaml
<?xml version="1.0" encoding="utf-8"?>
<Theme xmlns="http://wixtoolset.org/schemas/v4/thmutil">
	<Font Id="0" Height="-12" Weight="500" Foreground="windowtext" Background="window">Segoe UI</Font>

	<Window Width="485" Height="300" HexStyle="100a0000" FontId="0" Caption="Installer">
		<Page Name="Install">
			<RadioButtons Name="CHOICE" >
				<RadioButton Height="30" Width="30" X="33" Y="100" Value="1" />
				<RadioButton Height="30" Width="30" X="33" Y="130" Value="2" />
			</RadioButtons>
		</Page>
	</Window>
</Theme>
```

Versions: 
Wix 4.0.0-rc.2+a09eaf3c

WixToolset.Bal.wixext 4.0.0-rc.2
WixToolset.Util.wixext 4.0.0-rc.2

**Comment by barnson** [comment]
Start with an existing theme, then add your radio buttons, and only when that works, start removing stuff you don't need. WixStdBA assumes you have certain pages in your theme.

**Reply by ShakeyBones3** [reply]
Sorry, I had the entire standard theme and localization file. I made an [example repo here](https://github.com/ShakeyBones3/BundleAppExample) to show what I'm trying to do.

**Comment by rseanhall** [comment, marked as answer]
It's a bug - https://github.com/wixtoolset/issues/issues/7231. Also, you should add text to the radio buttons so the user knows what they're selecting.

**Reply by ShakeyBones3** [reply]
Awesome, thanks for the help!
